ReadMe.md

```diff
@@ -278,10 +284,14 @@ Conduction | 導電 | どうでん |
 Conductivity | 導電性 | どうでんせい |
 Connector | コネクタ | |
 Current | 電流 | でんりゅう |
+Daisy-chain | 橋渡し | はしわたし |
```

Comment:
I can't find examples of this via Google.
jisho.org mentions a different Japanese expression though: https://jisho.org/search/daisy%20chain

Reply:
It is used by japanese colleagues when talking about electronic devices daisy-chaining. Could you check with other EE japanese engineers to double-check ?

Reply:
If I knew any, I would :)

Reply:
removed from now, will look for more examples